tests/ClientBuilderTest.php

```diff
@@ -198,63 +199,13 @@ public function integrationsAreAddedToClientCorrectlyDataProvider(): array
         ];
     }
 
-    /**
-     * @expectedException \BadMethodCallException
-     * @expectedExceptionMessage The method named "methodThatDoesNotExists" does not exists.
-     */
-    public function testCallInvalidMethodThrowsException(): void
-    {
-        $clientBuilder = new ClientBuilder();
-        $clientBuilder->methodThatDoesNotExists();
-    }
-
-    /**
-     * @dataProvider optionsDataProvider
-     */
-    public function testCallExistingMethodForwardsCallToConfiguration(string $setterMethod, $value): void
-    {
-        $options = $this->createMock(Options::class);
-        $options->expects($this->once())
-            ->method($setterMethod)
-            ->with($this->equalTo($value));
-
-        $clientBuilder = new ClientBuilder();
-
-        $reflectionProperty = new \ReflectionProperty(ClientBuilder::class, 'options');
-        $reflectionProperty->setAccessible(true);
-        $reflectionProperty->setValue($clientBuilder, $options);
-        $reflectionProperty->setAccessible(false);
-
-        $clientBuilder->$setterMethod($value);
-    }
-
-    public function optionsDataProvider(): array
-    {
-        return [
-            ['setPrefixes', ['foo', 'bar']],
-            ['setSampleRate', 0.5],
-            ['setAttachStacktrace', true],
-            ['setContextLines', 0],
-            ['setEnableCompression', false],
-            ['setEnvironment', 'test'],
-            ['setExcludedProjectPaths', ['foo', 'bar']],
-            ['setExcludedExceptions', ['foo', 'bar']],
-            ['setProjectRoot', 'foo'],
-            ['setLogger', 'bar'],
-            ['setRelease', 'dev'],
-            ['setServerName', 'example.com'],
-            ['setTags', ['foo', 'bar']],
-            ['setErrorTypes', 0],
-        ];
-    }
-
     public function testClientBuilderFallbacksToDefaultSdkIdentifierAndVersion(): void
     {
         $callbackCalled = false;
         $expectedVersion = PrettyVersions::getVersion('sentry/sentry')->getPrettyVersion();
 
-        $clientBuilder = new ClientBuilder();
-        $clientBuilder->setBeforeSendCallback(function (Event $event) use ($expectedVersion, &$callbackCalled) {
+        $options = new Options();
```

Comment:
Instead of instantiating the `Options` object I would prefer to stick with the plain `array` of options